atlasdb-config/src/main/java/com/palantir/atlasdb/factory/DefaultLockAndTimestampServiceFactory.java

```diff
@@ -245,8 +223,9 @@ static LockAndTimestampServices createRawInstrumentedServices(
             Set<Schema> knownSchemas) {
         AtlasDbRuntimeConfig initialRuntimeConfig = runtimeConfig.get();
         assertNoSpuriousTimeLockBlockInRuntimeConfig(config, initialRuntimeConfig);
-        if (config.leader().isPresent()) {
-            return createRawLeaderServices(metricsManager, config.leader().get(), registrar, lock, time, userAgent);
+        if (config.leader().isPresent()) { // 1-node leader config
+            // equivalent to raw embedded; multi-node leader is not supported
+            return createRawEmbeddedServices(metricsManager, registrar, lock, time);
```

Comment:
Whichever PR merges last should remove `registrar` from this class

Reply:
I think I'm going to temporarily base my branch off yours to make it clear yours should merge first (and then be able to delete more from my PR)